build.sbt

```diff
@@ -1,11 +1,19 @@
+val scala2_12 = "2.12.15"
+val scala2_13 = "2.13.7"
+
 lazy val root = (project in file("."))
-  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
+  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning)
   .settings(
     majorVersion := 5,
     name := "performance-test-runner",
     makePublicallyAvailableOnBintray := true,
```

Comment:
one for PR bot - this should now be `isPublicRepo`

Reply:
Thanks I've changed `makePublicallyAvailableOnBintray` to `isPublicArtefact`